lib/index.js

```diff
@@ -71,14 +69,15 @@ function EslintValidationFilter(inputTree, options) {
   }
   this.options = options || {};
   this.eslintOptions = options.options || {};
-  // set inputTree
-  this.inputTree = inputTree;
+  // call base class constructor
+  Filter.call(this, inputTree);
 
   // set formatter
-  this.formatter = require(this.options.format || 'eslint/lib/formatters/stylish');
+  this.formatter = require(this.options.format || 'eslint/lib/formatters/stylish'); // eslint-disable-line global-require
 
   this.cli = new CLIEngine(this.eslintOptions);
 
+  this.eslintrc = inputTree;
```

Comment:
Also, this is going to be a bother, at least for the purpose of testing: eslint/eslint#5577